### PR TITLE
CUMULUS-2864: update ingestGranule not to perform separate validation request

### PR DESCRIPTION
**Pull request by jennyhliu**
**Summary:** Summary of changes

Addresses [CUMULUS-2864: Posting to CMR needlessly performs separate validation request](https://bugs.earthdata.nasa.gov/browse/CUMULUS-2864)

## Changes

* Detailed list or prose of changes
* ...

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: packages/cmr-client/src/CMR.ts [hunk 1]
```diff
@@ -272,31 +275,35 @@ export class CMR {
...
         {
           json: ummgMetadata,
           responseType: 'json',
           headers,
         }
       );
-      if (response.body.errors) {
-        throw new Error(`Failed to ingest, CMR Errors: ${response.body.errors}`);
-      }
+      return <CMRResponseBody>response.body;
```
Comment (flamingbear) [inline packages/cmr-client/src/CMR.ts:287]: There is no way the PUT can return a 200 with an error on the body? (the old code was wrong?)

File: packages/cmr-client/src/CMR.ts [hunk 2]
```diff
@@ -272,31 +275,35 @@ export class CMR {
...
+      }
+
+      log.error(errorMessage);
+
+      if (statusCode >= 500 && statusCode < 600) {
+        throw new CMRInternalError(errorMessage);
+      }
+
+      throw new Error(errorMessage);
+    }
```
Comment (flamingbear) [inline packages/cmr-client/src/CMR.ts:306]: 👍

File: packages/cmr-client/src/ingestConcept.ts [hunk 3]
```diff
@@ -50,17 +46,28 @@ async function ingestConcept(
...
-    return ingestResponseBody;
-  } catch (error) {
-    log.error(error, logDetails);
-    throw error;
+    log.error(errorMessage);
+
+    if (statusCode >= 500 && statusCode < 600) {
+      throw new CMRInternalError(errorMessage);
+    }
+    throw new Error(errorMessage);
```
Comment (flamingbear) [inline packages/cmr-client/src/ingestConcept.ts:70]: This is nice cleanup, I just can't find where got.put has to throw if it returns an error from the server. (same comment as previous change)

File: packages/cmr-client/tests/test-CMR.js [hunk 4]
```diff
@@ -165,6 +165,32 @@ test('getReadHeaders returns clientId and token for launchpad', (t) => {
   t.is(headers.Authorization, '12345');
 });
 
+test.serial.only('ingestUMMGranule() returns CMRInternalError when CMR is down', async (t) => {
```
Comment (flamingbear) [inline packages/cmr-client/tests/test-CMR.js:168]: remove `.only`

File: packages/cmr-client/src/CMR.ts (hunk 1 above)
Comment (flamingbear) [reply packages/cmr-client/src/CMR.ts:287]: Your test in test-CMR.js makes me think this is true.

**Review by flamingbear — CHANGES_REQUESTED** [review]
Everything looks good except for one .only to get removed 

I validated the behavior of `got.put` myself.

File: packages/cmr-client/src/ingestConcept.ts (hunk 3 above)
Comment (flamingbear) [reply packages/cmr-client/src/ingestConcept.ts:70]: I can't find it, but I validated it.

File: packages/cmr-client/src/CMR.ts (hunk 1 above)
Comment (flamingbear) [reply packages/cmr-client/src/CMR.ts:287]: Yup, I validated

**Review by flamingbear — APPROVED** [review]

**Comment by flamingbear** [comment]
looks good

File: example/cumulus-tf/ingest_and_publish_granule_with_orca_workflow.asl.json [hunk 5]
```diff
@@ -290,7 +290,6 @@
             "cmr": "{$.meta.cmr}",
             "launchpad": "{$.meta.launchpad}",
             "etags": "{$.meta.file_etags}",
-            "skipMetadataValidation": true,
```
Comment (jennyhliu) [inline example/cumulus-tf/ingest_and_publish_granule_with_orca_workflow.asl.json:293]: https://github.com/nasa/cumulus/pull/2892#discussion_r833934666

**Pull request merged**